reverse_proxy_samples/apache.conf

```
@@ -0,0 +1,10 @@
+# You can integrate this into your (virtual) host configuration or using
+# Debian you might put this to /etc/apache2/conf-available/mollysocket.conf
+# and use a2enconf mollysocket afterwards
+<Location /molly/>
+        ProxyPass http://localhost:8020/
+        ProxyPassReverse http://localhost:8020/
+        RequestHeader set X-Original-URL "expr=%{THE_REQUEST}"
+        RequestHeader edit* X-Original-URL ^[A-Z]+\s|\sHTTP/1\.\d$ ""
```

```suggestion
        RequestHeader set X-Original-URL "/molly/"
```

It is more efficient and less prone to errors/bugs to hard code it. For instance, what happens if we use HTTP/2 ?

---

Ok, I changed it.

---

Thank you ! Have you tested the new config and it works as expected ?

---

Yes, I viewed the QR-code using a web browser and used the UP-testing function (should be something like "test the configuration" in English)  in Molly's preferences.

---

Great !